README.rst

```diff
@@ -815,6 +832,23 @@ For more information about optional parameters check the
 It can be used to change the thresholds used by `TimeLoggingTestRunner <#openwisp_utilsteststimeloggingtestrunner>`_
 to detect slow tests (0.3s by default) and highlight the slowest ones (1s by default) amongst them.
 
+``OPENWISP_STATICFILES_VERSIONED_EXCLUDE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**default**: ``[]``
+
+Allows to pass a list of **Unix shell-style wildcards** for files to be excluded by `CompressStaticFilesStorage <#openwisp_utilsstorageCompressStaticFilesStorage>`_.
+
+Use Unix shell-style wildcards.
+
+Example usage:
+
+.. code-block:: python
+
+    OPENWISP_STATICFILES_VERSIONED_EXCLUDE = [
+    '*png',
```

Comment:
can you add 4 more spaces of indentation here please?

Reply:
yep sure 👍